cmd/imports/secrets.go

```diff
@@ -0,0 +1,87 @@
+package imports
+
+import (
+	"context"
+
+	"github.com/consensys/quorum-key-manager/pkg/errors"
+	"github.com/consensys/quorum-key-manager/pkg/json"
+	"github.com/consensys/quorum-key-manager/src/infra/log"
+	manifest "github.com/consensys/quorum-key-manager/src/infra/manifests/entities"
+	"github.com/consensys/quorum-key-manager/src/stores"
+	"github.com/consensys/quorum-key-manager/src/stores/database"
+	"github.com/consensys/quorum-key-manager/src/stores/entities"
+	"github.com/consensys/quorum-key-manager/src/stores/manager/secrets"
+)
+
+func ImportSecrets(ctx context.Context, db database.Secrets, mnf *manifest.Manifest, logger log.Logger) error {
+	logger.Info("importing secrets...", "store", mnf.Kind, "store_name", mnf.Name)
+
+	store, err := getSecretStore(mnf, logger)
+	if err != nil {
+		return err
+	}
+
+	storeIDs, err := store.List(ctx, 0, 0)
+	if err != nil {
+		return err
+	}
+
+	dbIDs, err := db.SearchIDs(ctx, false, 0, 0)
+	if err != nil {
+		return err
+	}
+
+	var n uint
+	for _, id := range difference(storeIDs, dbIDs) {
+		secret, err := store.Get(ctx, id, "")
+		if err != nil {
+			return err
```

Comment:
Maybe we should continue here and mark the failures ?

Reply:
added failure if not reachable